README.md

```diff
@@ -36,72 +36,104 @@ PID   USER     COMMAND
 
 ### OCF Container JSON Format:
 
-Below is a sample `container.json` configuration file. It assumes that
+Below is a sample `config.json` configuration file. It assumes that
 the file-system is found in a directory called `rootfs` and there is a
 user named `daemon` defined within that file-system.
 
 ```json
 {
-    "version": "0.1.1",
```

Comment:
why are all these changes here? i'm just using the output from `runc spec` and pasting it in the readme.